pkg/reconciler/contour/config/contour.go

```diff
@@ -49,18 +51,29 @@ type Contour struct {
 	DefaultTLSSecret      *types.NamespacedName
 	TimeoutPolicyResponse string
 	TimeoutPolicyIdle     string
+	CORSPolicy            *CORSPolicy
```

Comment:
Should just use the contour's cors struct

Reply:
we just switched to contour's struct